src/commonMain/kotlin/fr/acinq/eclair/db/PaymentsDb.kt

```diff
@@ -262,6 +259,11 @@ data class OutgoingPayment(val id: UUID, val recipientAmount: MilliSatoshi, val
     }
 }
 
+@Serializable
+enum class ChannelClosingType {
+    Mutual, Local, Remote, Revoked, Other;
```

Comment:
nit: what does the trailing `;` do?

Reply:
It just marks the end of the enum values, it is not needed in our case since the compiler is smart enough to know that from the `}` closing the enum. But if we were to add a method below the enum values, we would need a trailing `;` as a separator